### DISCUSSION TITLE
老毛子固件应该怎么设置？

### DISCUSSION THREAD

**Discussion by Pengkaniu** [natfrp/wiki#205] [使用问题] [answered]
自己尝试后无果，格式顺序和配置文件不一样

**Comment by fakeboboliu** [comment, marked as answer]
您好，

我们暂时并无使用此固件的硬件进行演示，也没有提供相关支持的计划。
但是我们已知其本质为 Linux 系统，
因此请搜索如何使用 ssh 登入后参考 [使用教程](https://doc.natfrp.com/#/frpc/usage) 的 Linux 用法即可。
